### ISSUE TITLE
Invalid use of FOV in Orthogonal view

### ISSUE THREAD

**Issue by gsohler** [openscad/openscad#4811]
**Describe the bug**
OpenSCAD displays and even reacts on FOV(field  of  view) on orthogonal view, whereas FOV is a perspective  parameter only.
I was not completely sure at 1st but looking at 
https://gamedev.stackexchange.com/questions/64423/orthographic-projection-with-variable-fov
states: 
Orthographic projection has no concept of a FOV, because by definition an orthographic viewing volume is cubic.

**To Reproduce**
1. Load your Model
2. Set your Viewing Mode to Orthogonal
3. View FOV parameter being displayed in the
4. Use Shift+Middle mouse button up and down watch, how FOV changes and even the model scales for Orthogonal view.

**Expected behavior**
FOV should not be displayed in orthogonal view, rather  a length scale how big "1" size of your model would display on your monitor
Yes, I appreciate the ability to scale my model with middle mouse button, but something is wrong here,  isn't it ?

**Code reproducing the issue**

**Screenshots**

**Environment and Version info (please complete the following information):**
Linux FC37
Latest master from 2023-11-04  compiled

**Comment by gsohler** [comment]
Having this fixed and having credible display this would actually be quite a good option to measure model dimensions when
the searched length is rotated to be parallel to the  display plane.

**Comment by jordanbrown0** [comment]
Indeed, I was just thinking about this.  Zoom is more obvious:  it works, and it does something zoom-y, but it does it by changing the distance... and why does that have any effect in orthogonal mode?

(Why was I thinking about it?  Because input devices like SpaceMice let you directly change things like the "translate" value, and in orthogonal mode changing the viewport-relative Y value has no effect.)

My mental model is that distance and FOV are combined to yield the world-coordinate size of a rectangle that is a ways in front of the camera, and that rectangle is orthogonally mapped onto the viewport.  This model explains why changing the viewport-relative-Y coordinate has no effect, because although that moves the centerpoint of the viewport space in and out, it doesn't change the distance and so doesn't change the size of that rectangle.

Part of the problem is that in orthogonal mode the entire notion of a camera is incorrect.  In orthogonal mode there is no one point from which the viewer is looking at the model, with view lines radiating out from that point; rather, there is a finite plane (the viewport) through which the viewer is looking out at the model along parallel lines.

Perhaps in orthogonal mode the definitions of the controls should be subtly different and the status displayed slightly different, to achieve basically the same effect but do it through more directly obvious values.
